lib/setting-api-key.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Add the Google Fonts API key setting to the DPP settings tab.
+ *
+ * @package gppro-google-webfonts
+ */
+
+/**
+ * Add the Google Webfonts API Key setting.
+ *
+ * @param array $settings Existing settings.
+ *
+ * @return array
+ */
+function gppro_google_webfonts_api_key( $settings ) {
+	$description = sprintf(
+		'%1$s <a href="%2$s" target="_blank">%3$s</a> %4$s.',
+		__( 'You must have a', 'gppro-google-webfonts' ),
```

Comment:
this should really be concatenated via sprintf with placeholders in the since string instead of separate strings like this for ease of translation.

Reply:
@NicktheGeek this is updated, but I'm not sure if I did it right.